pyiceberg/table/__init__.py

```diff
@@ -1170,14 +1174,18 @@ def _(
 
 
 def update_table_metadata(
-    base_metadata: TableMetadata, updates: Tuple[TableUpdate, ...], enforce_validation: bool = False
+    base_metadata: TableMetadata,
+    updates: Tuple[TableUpdate, ...],
+    enforce_validation: bool = False,
+    metadata_location: Optional[str] = None,
```

Comment:
Is `base_metadata.location` the same as `metadata_location`?

Reply:
No, the `location` property in metadata indicates table's base location. This is used by writers to determine where to store data files, manifest files, and table metadata files.

Reply:
I see, thanks for the clarification!